lib/msf/ui/console/driver.rb

```diff
@@ -178,6 +178,22 @@ def initialize(prompt = DefaultPrompt, prompt_char = DefaultPromptChar, opts = {
       }
     end
 
+    # Process persistent job handler
+    begin
+      restore_handlers = JSON.parse(File.read(Msf::Config.persist_file))
+    rescue Errno::ENOENT, JSON::ParserError
+      restore_handlers = nil
+    end
+
+    unless restore_handlers.nil?
```

Comment:
For next time, double negatives can be confusing. I think `if restore_handlers` would suffice.